GOVERNANCE.md

```diff
@@ -0,0 +1,86 @@
+# Mui-Org Governance
+
+### Overview
```

Comment:
I'm wondering, what about following the `#`>`##` hierarchy?

Reply:
Thanks @oliviertassinari - I fixed the hierarchy for all the headings. 👍